### DISCUSSION TITLE
Using FontAwesome icons in Taipy GUI

### DISCUSSION THREAD

**Discussion by CDucloux** [Avaiga/taipy#2033] [Q&A]
I was wondering if there was a way to include FontAwesome icons in Taipy GUI ? 

For instance, in the markdown below : 

```python
markdown = """
# Generated by *Taipy*

You can insert *Markdown* code in a `text` control to
add style to the text.

If a line ends with two white spaces, such as here  
then you can create line skips.

Can you then somehow insert an icon ?
"""
```

**Comment by FlorianJacta** [comment]
You should be able to insert icons when copying from somewhere. Have you tried? However, I am not sure we support FontAwesome 

See an icon here in the title of the app: https://sales-dashboard.taipy.cloud/

**Reply by FlorianJacta** [reply]
Would this fix your issue? If you want a better integration, you can create an issue on this repo and we could add it into our roadmap!
